contribution/contributions.md

```diff
@@ -146,8 +146,13 @@ We are following Google Code style:
 * [Java style](https://google.github.io/styleguide/javaguide.html)
 * [Shell style](https://google.github.io/styleguide/shell.xml)
 
-Check style report location are in `${submodule}/target/site/checkstyle.html`
-Test coverage report location are in `${submodule}/target/site/cobertura/index.html`
+There are some plugins to format, lint your code in IDE (use [_tools/checkstyle.xml](https://github.com/apache/zeppelin/tree/master/_tools) as rules)
+
+* [Checkstyle plugin for Intellij](http://stackoverflow.com/questions/26955766/intellij-idea-checkstyle)
```

Comment:
@1ambda 
I'm using OSX and the answer in this stackoverflow question doesn't fit for me(it's based on Windows version and 2014 ver I guess). Since I can't access `File -> Settings` to set checkstyle in my IntelliJ 2016 for macOS. How about the official link provided from Jetbrains? I think it's more reliable. 
https://plugins.jetbrains.com/plugin/1065

![screenshot](https://cloud.githubusercontent.com/assets/10060731/20695602/a229bd7a-b630-11e6-9317-567e0893ad12.png)

The answer also has this link :)